storage/innobase/xtrabackup/test/t/check_for_passwords.sh

```diff
@@ -28,7 +28,8 @@ function grep_in_datadir_ps() {
   ### to the table and waits for the running update on MyISAM to finish.
   mysql -e "SELECT SLEEP(1000) from t1 for update " test &
 
-  $XB_BIN $XB_ARGS --backup -u pxb $additional_options\
+  ### lock-ddl=false is required since LTFB will wait for update on MyISAM table to finish
+  $XB_BIN $XB_ARGS --backup -u pxb $additional_options --lock-ddl=false \
```

Comment:
@satya-bodapati @altmannmarcelo , I had to use one more hack to make this test release build. for PS LTFB is acquired before generated xtrabackup_log file. 
Let me know if you fine with it else I will make this test debug only.

Reply:
isn't the intention for backup to wait until the query is killed?

Reply:
yes the intention is to let backup wait.  Backup will still wait with --lock-ddl=false. It just it won't acquire LTFB at early stage.

Reply:
does it (LTFB) happens twice? from where are these request for locks initiated?

Reply:
yes for for PS it happens twice. at the end instead of flush tables for MyISAM it do LTFB.